driver/SHiELD/atmosphere.F90

```diff
@@ -670,6 +672,8 @@ subroutine atmosphere_control_data (i1, i2, j1, j2, kt, p_hydro, hydro, tile_num
    if (present(p_hydro)) p_hydro = Atm(mygrid)%flagstruct%phys_hydrostatic
    if (present(  hydro))   hydro = Atm(mygrid)%flagstruct%hydrostatic
    if (present(tile_num)) tile_num = Atm(mygrid)%global_tile
+   if (present(do_inline_mp)) do_inline_mp = Atm(mygrid)%flagstruct%do_inline_mp
+   if (present(do_cosp)) do_cosp = Atm(mygrid)%flagstruct%do_cosp
```

Comment:
This code will not compile without including other not-yet-released updates. I would like to not merge this in and instead bring this over with our release.  Which I expect to be ready soon.